core/src/components/range/range.scss

```diff
@@ -308,3 +308,24 @@
   min-width: 100px;
   max-width: 200px;
 }
+
+// Range Label Placement - Stacked
+// ----------------------------------------------------------------
+
+/**
+ * Label is on top of the range.
+ */
+:host(.range-label-placement-stacked) .range-wrapper {
+  flex-direction: column;
+
+  align-items: normal;
+}
+
+:host(.range-label-placement-stacked) .label-text-wrapper {
```

Comment:
The pin on a range with a stacked label currently overlaps the text. We add padding to the host `.range-has-pin` is present, but we should probably add that as bottom margin to the label so the pin never obscures the label.

At the very least, you'll need to add bottom margin to the `.label-text-wrapper` element in each mode stylesheet. You can use the existing padding values that we set on the host when the `.range-has-pin` class is present in each mode stylesheet.

We then need to remove the host padding since we are setting it as margin on the label wrapper instead. There are a couple ways you can do this:

1. Set `@include padding(0px)` in the base stylesheet when `.range-label-placement-stacked` is present.
2. Update the existing `.range-has-pin` selectors to exclude `.range-label-placement-stacked` using `:not`.

We'll also want to capture a screenshot for this.